### DISCUSSION TITLE
ComboBox searching not works properly while using Japanese or Chinese keyboard layout

### DISCUSSION THREAD

**Discussion by sudharsan-narayanan** [microsoft/microsoft-ui-xaml#9425] [General]
### Describe the bug
ComboBox searching not works properly while using Japanese or Chinese keyboard layout. On further while using Japanese or Chinese keyboard ComboBox PreviewKeyDown does not fire. Please find the sample and video for further reference.

Sample:
[ComboBox-Searching-issue.zip](https://github.com/microsoft/microsoft-ui-xaml/files/11564761/ComboBox-Searching-issue.zip)

Video:
[ComboBox-searching-issue.zip](https://github.com/microsoft/microsoft-ui-xaml/files/11564958/ComboBox-searching-issue.zip)

### Steps to reproduce the bug
Run the sample.
Open ComboBox drop-down.
Now, type "devel" note developer item was not highlighted.
Expected behavior
ComboBox searching should works properly while using Japanese or Chinese keyboard layout

### Screenshots
No response

### NuGet package version
WinUI 3 - Windows App SDK 1.3.1: 1.3.230502000

### Windows version
Windows 11 (22H2): Build 22621

### Additional context
No response

**Comment by codendone** [comment]
Note: This is also covered by #8505.

**Reply by sudharsan-narayanan** [reply]
Hi @codendone,

Any update on this issue? reported on [#8505](https://github.com/microsoft/microsoft-ui-xaml/issues/8505)

**Reply by codendone** [reply]
I see you pinged in that issue in the last day and I've forwarded that ping to the area owner.
